system_files/shared/usr/bin/check-sb-key

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+FINGERPRINT="$(openssl x509 -fingerprint -noout -in /etc/pki/akmods/certs/akmods-ublue.der | cut -c 18-)"
+mokutil --list-enrolled | grep -q $FINGERPRINT
```

Comment:
This doesn't work as expected. The current command will always report that the key is not enrolled.

This is due to the output from `mokutil --list-enrolled` vs `mokutil --list-enrolled --verbose-listing`:
```
$ mokutil --list-enrolled 
2bb010e24d fedoraca
2be991e3b1 ublue kernel
```
The output above will never have the fingerprint contents.

Adding `--verbose-listing` adds the fingerprint output to the listing, however, with a lowercased fingerprint, which won't still won't match what you have here.
```
$ mokutil --list-enrolled --verbose-listing | grep -i sha1
SHA1 Fingerprint: 2b:b0:10:e2:4d:94:c6:32:24:58:89:ba:aa:9e:d0:f3:d5:ef:1f:68
SHA1 Fingerprint: 2b:e9:91:e3:b1:b5:40:70:f4:3d:80:bb:13:eb:c6:57:e5:a3:78:0d
```


Example of a working match:
```
$ mokutil --list-enrolled --verbose-listing | grep -i $FINGERPRINT
SHA1 Fingerprint: 2b:e9:91:e3:b1:b5:40:70:f4:3d:80:bb:13:eb:c6:57:e5:a3:78:0d
```

Which, converted to work in the script would be:
```
mokutil --list-enrolled --verbose-listing | grep -q -i $FINGERPRINT
```